src/mumble/LookConfig.ui

```diff
@@ -730,117 +760,126 @@
-        <property name="toolTip">
-         <string>Relative font size to use in the Talking UI in percent.</string>
-        </property>
-        <property name="text">
-         <string>Rel. font size (%)</string>
-        </property>
-        <property name="buddy">
-         <cstring>qsbRelFontSize</cstring>
+         <string>Postfix character count</string>
         </property>
        </widget>
       </item>
-      <item row="8" column="0">
-       <widget class="QLabel" name="qlChannelHierarchyDepth">
+      <item row="7" column="1">
+       <widget class="QSpinBox" name="qsbChannelHierarchyDepth">
         <property name="toolTip">
          <string>The names of how many parent channels should be included in the channel's name when displaying it in the TalkingUI?</string>
         </property>
-        <property name="text">
+        <property name="accessibleName">
          <string>Channel hierarchy depth</string>
         </property>
-        <property name="buddy">
-         <cstring>qsbChannelHierarchyDepth</cstring>
-        </property>
        </widget>
       </item>
-      <item row="0" column="0" colspan="2">
-       <widget class="QCheckBox" name="qcbUsersAlwaysVisible">
-        <property name="toolTip">
-         <string>If this is checked, users will always be visible in the TalkingUI (regardless of talking state).</string>
-        </property>
-        <property name="text">
-         <string>Always keep users visible</string>
-        </property>
-       </widget>
+      <item row="12" column="1">
+       <layout class="QHBoxLayout" name="horizontalLayout">
+        <item>
+         <widget class="QPushButton" name="qbBackgroundColor">
+          <property name="sizePolicy">
+           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
+            <horstretch>0</horstretch>
+            <verstretch>0</verstretch>
+           </sizepolicy>
+          </property>
+          <property name="toolTip">
+           <string>Color that gets used for the background of the talkingUI.</string>
+          </property>
+          <property name="accessibleName">
+           <string>Background Color</string>
+          </property>
+          <property name="text">
+           <string>Choose Color</string>
+          </property>
+         </widget>
+        </item>
+        <item>
+         <widget class="QFrame" name="qccolorPreview">
+          <property name="minimumSize">
+           <size>
+            <width>50</width>
+            <height>0</height>
+           </size>
+          </property>
+          <property name="styleSheet">
+           <string notr="true">background-color: white</string>
+          </property>
+          <property name="frameShape">
+           <enum>QFrame::Shape::Box</enum>
+          </property>
+          <property name="frameShadow">
+           <enum>QFrame::Shadow::Raised</enum>
+          </property>
+         </widget>
+        </item>
+       </layout>
```

Comment:
Your new button breaks the tab order. If you are using QtCreator go to "Edit" -> "Edit Tab Order"
The label of your new button is also not setting the ``buddy`` attribute correctly, it should be your new button. In QtCreator there is "Edit" -> "Edit buddies"

Otherwise, from an accessibility perspective your change seems fine. Just to be sure please double-check the [accessibility checklist](https://github.com/mumble-voip/mumble/blob/master/docs/dev/Accessibility.md#checklist-for-ui-changes-in-mumble)